packages/core/src/oidc/adapter.ts

```diff
@@ -15,6 +15,11 @@ import {
 
 import { getApplicationTypeString } from './utils';
 
+export enum GrantType {
+  AuthorizationCode = 'authorization_code',
+  RefreshToken = 'refresh_token',
+}
```

Comment:
Don't we have this in schemas? If not, then would it be nice to put this enum in schemas, so that it can be re-used across all logto sub-packages

Reply:
Moved. PTAL